app.js

```diff
@@ -0,0 +1,56 @@
+// declare variables we need
+const scoreInput = document.querySelector("#score");
+const diceBox = document.querySelector("#dice-box");
+const rollBtn = document.querySelector("#roll-button");
+const playerStatus = document.querySelector("#player-status");
+const diceImg = document.querySelector("#dice-img");
+let score = 0;
+
+diceBox.style.display = "none";
+
+// event listener triggered on button click with condition
+rollBtn.addEventListener("click", () => {
+   if (rollBtn.textContent === "Roll") {
+      let randomNumber = Math.floor(Math.random() * 6 + 1);
+      checkNumber(randomNumber);
+   } else if (rollBtn.textContent === "Start again") {
+      resetGame();
+   }
+})
+
+// function to reset game
+const resetGame = () => {
+   if (rollBtn.textContent === "Start again") {
+      score = 0;
+      scoreInput.textContent = "0";
+      rollBtn.textContent = "Roll";
+      playerStatus.textContent = "Player 1";
+      diceBox.style.display = "none";
+   }
+}
+
+// function to apply condition to generated random number
+const checkNumber = (randomNumber) => {
+   if (randomNumber === 1) {
+      diceBox.style.display = "block";
+      diceImg.src = "img/dice1.png";
```

Comment:
Even though 1 is a special case here, the number is still stored in random number so you can use line 44 and move it from line 44 outside the if/else and remove some more duplicated code :)